src/devdb/devdb.py

```diff
@@ -6,6 +6,160 @@ def devdb():
     import pdb
     from src.devdb.helpers import get_data
     from src.devdb.components.qaqc_app_report import QAQCAppReport
+    from src.devdb.components.qaqc_version_history_report import (
+        QAQCVersionHistoryReport,
+    )
+
+    QAQC_CHECK_GROUPS = {
+        "Class B": "These checks are related to class A and class B unit distinctions.",
+        "Units": "These checks are related to missing or suspicious unit counts.",
+        "Duplicates": "These checks are related to identifying possible duplicates",
+        "Geography": "These checks are related to potentially missing geographic information",
+        "Other/Unique": "Miscellaneuous Checks",
+        "Outlier": "These checks are related to outlying or unexpected values",
+    }
+    QAQC_CHECK_DICTIONARY = {
+        "b_likely_occ_desc": {
+            "description": "Likely Class B Units: OCC Initial or Proposed contains hotel, assisted, etc. or job description contains Hotel, Motel, etc.",
+            "field_type": "boolean",
+            "group": "Class B",
```

Comment:
If I'm understanding correctly, this key associates the variable with the plot it's displayed in right? Would renaming the key to "section" be in line with the terminology we decided on?

Reply:
Yeah that makes sense